site/docs/tasks/custom-taxonomy.md

```diff
@@ -0,0 +1,42 @@
+# Using a Custom Taxonomy for Resource Validation
+
+The project defines a set of immutable structural JSON schemas, or "taxonomies" for resources deployed in Fybrik. 
+However, since the taxonomy is meant to be configurable, a `taxonomy.json` file is referenced from these schemas for any definition that is customizable.
+
+The `taxonomy.json` file is generated from a base taxonomy and zero or more taxonomy layers:
+
+- The base taxonomy is maintained by the project and includes all of the structural definitions that are subject to customization (e.g.: tags, actions). 
+
+- The taxonomy layers are maintained by users and external systems that add customizations over the base taxonomy (e.g., defining specific tags, actions).
+
+
+This task describes how to deploy Fybrik with a custom `taxonomy.json` file that is generated with the Taxonomy Compile CLI tool. 
+
+## Taxonomy Compile CLI tool 
+A CLI tool for compiling a base taxonomy and zero or more taxonomy layers is provided in our repo. 
+The following command can be used from the root directory of our repo to run this tool. 
+
+Usage:
+```bash
+  go run main.go taxonomy compile --out <outputFile> --base <baseFile> [<layerFile> ...] [--codegen]
+```
```

Comment:
perhaps give an example of the baseFile or a path in the repo where such is placed.

Reply:
added links to our base.yaml and example layers in the documentation